### ISSUE TITLE
MODULE_NOT_FOUND error while trying to load @upscalerjs/esrgan-thick/4x (or 2x, or any other model)

### ISSUE THREAD

**Issue by jeffcrouse** [thekevinscott/UpscalerJS#1114]
**Describe the bug**
MODULE_NOT_FOUND error while trying to load @upscalerjs/esrgan-thick/4x
Error: Cannot find module '@upscalerjs-for-esbuild/esrgan-thick'
Even though @upscalerjs/esrgan-thick is installed

**To Reproduce**
I first noticed the issue on my Windows machine, but it also occurs on codesandbox
[https://codesandbox.io/p/sandbox/hopeful-kapitsa-hw7fc3?file=%2Findex.js%3A1%2C1](https://codesandbox.io/p/sandbox/hopeful-kapitsa-hw7fc3?file=%2Findex.js%3A1%2C1)

**Expected behavior**
The image should be upscaled using the 4x model

**Screenshots**
![Screenshot 2023-08-23 130959](https://github.com/thekevinscott/UpscalerJS/assets/15384/e0f7e06f-5bc2-4021-af51-62e9e07aa6ff)

**Additional context**
This worked perfectly for me with these versions...
"@tensorflow/tfjs-node": "^4.8.0",
"@upscalerjs/esrgan-thick": "^1.0.0-beta.11",
"upscaler": "^1.0.0-beta.16",

These are the ones that don't work:
 "@tensorflow/tfjs-node": "^4.10.0",
"@upscalerjs/esrgan-thick": "^1.0.0-beta.12",
"upscaler": "^1.0.0-beta.17"

So perhaps it's something that happened between these versions?

**Comment by jeffcrouse** [comment]
UPDATE:
Disregard what I said above about previous versions working -- I downgraded and I am still having the issue.  
It _did_ work for me at some point, which would suggest that it's something about my environment.  
However, I have the same issue on codesandbox, 
I've made a much more minimal example here: https://codesandbox.io/p/sandbox/eloquent-river-xrdwh4

**Comment by thekevinscott** [comment]
Thank you very much for reporting this.

I've just published a new version, `1.0.0-beta.13`. Give it a try and see if it works. [Here's a codesandbox that should be working](https://codesandbox.io/p/sandbox/recursing-alex-kls8jw?file=%2Fpackage.json%3A26%2C41).

The longer answer is there's a bug in the publish script, that only manifests itself on Node (not in the browser). So I've manually published the version and will put better infrastructure in place.

Let me know if it's working for you now.

**Comment by thekevinscott** [comment]
(Codesandbox is crashing for me with an OOM issue; maybe give it a shot locally first just to be safe)

**Comment by jeffcrouse** [comment]
Yep, working for me now.  Thanks so much for providing this package!
